include/seqan3/alignment/configuration/utility.hpp

```diff
@@ -173,12 +175,13 @@ inline constexpr align_cfg::id align_config_type_to_id_v = align_config_type_to_
 inline constexpr std::array<std::array<bool, static_cast<uint8_t>(align_cfg::id::SIZE)>,
                             static_cast<uint8_t>(align_cfg::id::SIZE)> align_config_validation_matrix =
 {
-    //  gap    s_ends score  global output
-    { { false, true,  true,  true,  true  },  // gap
-      { true,  false, true,  true,  true  },  // sequence_ends
-      { true,  true,  false, true,  true  },  // score
-      { true,  true,  true,  false, true  },  // global
-      { true,  true,  true,  true,  false } } // output
+    //  gap    s_ends score  global output  m_err
+    { { false, true,  true,  true,  true , true },  // gap
+      { true,  false, true,  true,  true , true },  // sequence_ends
+      { true,  true,  false, true,  true , true },  // score
+      { true,  true,  true,  false, true , true },  // global
+      { true,  true,  true,  true,  false, true },  // output
+      { true,  true,  true,  true,  false, false} } // max_error
```

Comment:
does that mean that `max_error` can't combined with `output`?

Reply:
only if output was set before max_error.
I find this kind of logical. 🤔

Reply:
Good spot 🤣

Reply:
What do you mean by "good spot"?